src/main/java/qna/domain/Question.java

```diff
@@ -54,46 +60,61 @@ public boolean isOwner(User writer) {
 
     public void addAnswer(Answer answer) {
         answer.toQuestion(this);
+        this.answers.add(answer);
+    }
+
+    public DeleteHistories deleteByWriterAndThenCallbackDeleteHistories(User writer) {
```

Comment:
```suggestion
    public DeleteHistories delete(User writer) {
```

return 타입과 서비스의 정책으로 의미를 추론할 수 있어서 메서드 내용에 모든 내용을 작성하지 않으셔도 될 것 같은데 어떻게 생각하시나요?

Reply:
말씀대로 바꾸는게 좋을거 같습니다!

내용을 넣어준 이유는 `delete` 행위를 하는데 무언가를 리턴한다는건 일반적인 행동은 아니라는 생각에 명시해줄 필요성이 있겠다는 생가이었습니다.